types/instance.go

```diff
@@ -70,3 +80,75 @@ func (i Instance) GetItem() interface{} {
 func (i Instance) GetType() string {
 	return "instance"
 }
+
+type JsonResponseBody struct {
+	Json map[string]interface{}
```

Comment:
As i see nobody uses the json response, so it just generates complexity. What do you think?

Reply:
This is where I've got some private code that implements business logic for our use case. So, this isn't exactly the same definition as what I've got internally.

Secondly, checking the response of a REST API is often context-specific. So, consumers of the JSON response will need to cast this interface into whatever types they've defined for that particular response.

So, I agree with you that this is complexity that is not currently in use in the project. It's a stub for future work to build on top of.

Now that I've shared my rationale, please confirm that you'd still like it removed. 🙂

Reply:
Thanks i see now. My suggestion is to move json marshalling into the private codebase. In this way each dependent project should be able to convert/cast byte body to any custom format not just `map[string]interface{}`.